n_helpers.c

```diff
@@ -2341,11 +2341,10 @@ bool NoteDebugSyncStatus(int pollFrequencyMs, int maxLevel)
         J *body = JGetObject(rsp, "body");
         if (body != NULL) {
             if (maxLevel < 0 || JGetInt(body, "level") <= maxLevel) {
-                _Debug("sync: ");
-                _Debug(JGetString(body, "subsystem"));
-                _Debug(" ");
-                _Debug(JGetString(body, "text"));
-                _Debug("\n");
+                _DebugWithLevel(0, "[SYNC] ");
```

Comment:
Should these 0s be `NOTE_C_LOG_LEVEL_ERROR`?

Reply:
Yeah, but it's not an error.

I was thinking it would be better to create an alias `NOTE_C_LOG_LEVEL_ALWAYS`, but wanted to get your opinion first.

Reply:
I would use `NoteDebug`. Seems like there's no point in using the version of the function with log level if you want to always log it.

Reply:
Yeah, that's a good point. I guess I really like the expressiveness of `NoteDebugWithLevel()`, and I hadn't even considered using the standard logging.